### PR TITLE
Add Emilyhammer burbank file with recs

### LINKED ISSUE #17: Create a file for emilyhammer Burbank
- [ ] Create a branch
- [ ] Add commit 
- [ ] Open a PR
- [ ] Have a discussion
- [ ]  Seek approval
- [ ]  Merge PR

### PR DESCRIPTION
**Pull request by emilyhammer**
closes #17

### CODE REVIEW

**Comment by githubteacher** [comment]
:octocat: 😄 👋

File: emilyhammer-Burbank.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Burbank, CA
+
+### Places to Eat
+- Betos
+- Lure
+
+### Places to Go
```
Comment (githubteacher) [inline emilyhammer-Burbank.md]: Why didn't you add Harry Potter land?

File: emilyhammer-Burbank.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- Disney
+- Universal Studios
```
Comment (githubteacher) [inline emilyhammer-Burbank.md]: I guess this counts.

File: emilyhammer-Burbank.md (hunk 1 above)
Comment (emilyhammer) [reply emilyhammer-Burbank.md]: Added

File: emilyhammer-Burbank.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Burbank, CA
+
+### Places to Eat
+- Betos
+- Lure
+- McDonalds
```
Comment (githubteacher) [inline emilyhammer-Burbank.md:6]: 😭

File: emilyhammer-Burbank.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+- Burger King
```
Comment (githubteacher) [inline emilyhammer-Burbank.md:7]: NOOOOOOOOOO

**Review by githubteacher — APPROVED** [review]
This is pretty great. I'll have to check out this "McDonalds" place. 🍟

**Pull request merged**